inst/rmarkdown/templates/xaringan/resources/useR.css

```diff
@@ -1,6 +1,6 @@
 .remark-slide-content {
-  background-image: url("https://www.r-project.org/conferences/useR.png"), linear-gradient(to bottom, white 85%, rgba(227, 227, 227) 95%) ;
-  background-position: 8% 95% , center;
+  background-image: url("https://www.dropbox.com/s/ff7nr4n2dcm59de/useR_logo.png?dl=1"), linear-gradient(to bottom, white 85%, rgba(227, 227, 227) 95%) ;
```

Comment:
I'd suggest using a URL that may be more "permanent" than a Dropbox URL (since Dropbox files may be moved or deleted, and you may forget to update the URL here), but I'll let you decide. If you want to do that, please feel free to submit another PR.